source/clear-linux/clear-linux.rst

```diff
@@ -3,26 +3,38 @@
 Clear Linux\* Project for Intel® Architecture
```

Comment:
@krisdale.  This looks great. I tested additional :hidden: directive on toc, and it worked fine. Just one minor edit: Please change title of document to match updated project name--based on our conversation today.

Reply:
Note: We agree to revise title after comprehensive review of docs per new Project Name. Therefore, I approved this PR.